packages/tables/resources/views/components/reorder/trigger.blade.php

```diff
@@ -0,0 +1,10 @@
+@props([
+    'enabled' => false,
+])
+
+<x-tables::icon-button
+    wire:click="toggleTableReordering"
+    :icon="$enabled ? 'heroicon-o-arrow-left' : 'heroicon-o-selector'"
```

Comment:
Maybe the check icon would look better here instead of the arrow?

```suggestion
    :icon="$enabled ? 'heroicon-o-check' : 'heroicon-o-selector'"
```